packages/nocodb/src/lib/db/sql-mgr/v2/SqlMgrv2.ts

```diff
@@ -118,7 +118,7 @@ export default class SqlMgrv2 {
     return sqlMigrationStatements;
   }
 
-  protected getSqlClient(base: Base) {
-    return NcConnectionMgrv2.getSqlClient(base);
+  protected async getSqlClient(base: Base): Promise<any> {
+    return await NcConnectionMgrv2.getSqlClient(base);
```

Comment:
`await` not required here since `NcConnectionMgrv2.getSqlClient(base)` returns a promise